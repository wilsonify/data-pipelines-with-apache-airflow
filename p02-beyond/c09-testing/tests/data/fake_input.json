[
  {
    "name": "bob",
    "age": "41",
    "sex": "M"
  },
  {
    "name": "alice",
    "age": "24",
    "sex": "F"
  },
  {
    "name": "carol",
    "age": "60",
    "sex": "F"
  }
]