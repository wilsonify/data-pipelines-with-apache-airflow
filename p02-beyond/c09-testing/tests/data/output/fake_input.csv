age,name,sex
41,bob,M
24,alice,F
60,carol,F
