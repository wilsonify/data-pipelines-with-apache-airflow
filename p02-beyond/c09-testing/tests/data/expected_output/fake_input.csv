age,sex,name
41,M,bob
24,F,alice
60,F,carol
